src/TEOS10.jl

```diff
@@ -0,0 +1,305 @@
+module TEOS10
+
+export 
+    TEOS10SeawaterPolynomial,
+    TEOS10EquationOfState
+
+using SeawaterPolynomials: AbstractSeawaterPolynomial, BoussinesqEquationOfState
+
+import SeawaterPolynomials: ρ′, thermal_expansion, haline_contraction
+
+#####
+##### The TEOS-10 polynomial approximation implemented in this file has been translated
+##### into Julia from https://github.com/fabien-roquet/polyTEOS/blob/master/polyTEOS10.py
+#####
+
+struct TEOS10SeawaterPolynomial{FT} <: AbstractSeawaterPolynomial end
+
+TEOS10SeawaterPolynomial(FT=Float64) = TEOS10SeawaterPolynomial{FT}()
+
+const EOS₁₀ = BoussinesqEquationOfState{<:TEOS10SeawaterPolynomial}
+
+TEOS10EquationOfState(FT=Float64; reference_density=1024.6) =
```

Comment:
Where did 1024.6 come from? Might be good to cite a source or explanation.

Reply:
It came out of thin air, just like 1020 !